src/redux/store.js

```diff
@@ -0,0 +1,10 @@
+import { configureStore } from '@reduxjs/toolkit';
+import booksReducer from './books/booksSlice';
+
+const store = configureStore({
+  reducer: {
+    book: booksReducer,
+  },
+});
+
+export default store;
```

Comment:
Nice job adding the bookReducer to your store👍. currently, you haven't added the categoriesReducer😉. Kindly add it as this is a requirement in this milestone👍.